examples/components/MNIST/preprocessing/run.py

```diff
@@ -117,13 +118,35 @@ def preprocess_data(
     test_dataset = MnistDataset(X_test.float(), y_test, transform=transform)
     datasets = {"train": train_dataset, "test": test_dataset}
 
+    # Mlflow logging
+    log_metdata(X_train, X_test)
```

Comment:
typo: log_metadata

Reply:
also, could we also use metrics_prefix here to display the metadata under a metric name that shows which silo it is?

Reply:
sure, did the required changes.